target/step13-2020-1/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why is the target folder name "step13-2020-1"?

and not just "step13-2020"

Reply:
It must be named that for the pom.xml file to work. The folder format stands for the project name and the version number. When Maven is run, it needs to search for the Web XML files in a folder of that format.